app/core/Tracker.coffee

```diff
@@ -89,8 +89,8 @@ module.exports = class Tracker extends CocoClass
 
     # Mixpanel
     # https://mixpanel.com/help/reference/javascript
-    mixpanel.identify(me.id)
-    mixpanel.register(traits)
+    mixpanel?.identify(me.id)
+    mixpanel?.register(traits)
```

Comment:
We could nuke it at some point, I was leaving open the possibility wed put it back in just for teachers or load it some other way.